test/python/Dockerfile

```diff
@@ -247,6 +258,21 @@ USER 1000
 # renovate: datasource=pypi
 RUN install-tool pdm 2.15.3
 
+#--------------------------------------
+# test: rye: rye (multiple python)
```

Comment:
? i don't see multiple python. should be rootless indstead?

Reply:
Ah, yes, indeed. will fix asap

Reply:
Also, Rye can be configured to fetch a specific version of the python interpreter based on the pyproject file it encounters. This is the default setting. As there's actually no dependency on python at all. I just put it under python since that's what its intended purpose is.